docs/excel/excel-data-types-entity-card.md

```diff
@@ -0,0 +1,173 @@
+
+This article describes how to use the [Excel JavaScript API](../reference/overview/excel-add-ins-reference-overview.md) to work with the card component of entity value data types. An entity value, or [EntityCellValue](/javascript/api/excel/excel.entitycellvalue), is a container for data types and similar to an object in object oriented programming.
+
+The card component displays as a modal window in the Excel UI and can show additional information contained within an entity value, beyond what's already visible in a cell. This article introduces card properties, layout options for the card, and card data attribution functionality.
+
+The following screenshot shows an example of an open entity value card, in this case for the **Tofu** product from a list of grocery store products.
+
+:::image type="content" source="../images/excel-data-types-entity-card-tofu.png" alt-text="A screenshot showing an entity value data type with the card window displayed.":::
+
+## Card properties
+
+The entity value [`properties`](/javascript/api/excel/excel.entitycellvalue#excel-excel-entitycellvalue-properties-member) property allows you to set customized information about your data types. The `properties` key accepts nested data types. Each nested property, or data type, must have a `type` and `basicValue` setting.
+
+> [!IMPORTANT]
+> The nested `properties` data types are used in combination with the [Card layout](#card-layout) values described in the subsequent article section. After defining a nested data type in `properties`, it must be assigned in the `layouts` property to display on the card.
+
+The following code snippet shows the JSON for an entity value with multiple data types nested within `properties`.
+
+> [!NOTE]
+> The following code snippet is an excerpt. To see the complete code sample, visit the [OfficeDev/office-js-snippets](https://github.com/OfficeDev/office-js-snippets/blob/main/samples/excel/85-preview-apis/data-types-entity-values.yaml) repository.
+
+```json
+const entity: Excel.EntityCellValue = {
+    type: Excel.CellValueType.entity,
+    text: productName,
+    properties: {
+        "Product ID": {
+            type: Excel.CellValueType.string,
+            basicValue: productID.toString() || ""
+        },
+        "Product Name": {
+            type: Excel.CellValueType.string,
+            basicValue: productName || ""
+        },
+        "Quantity Per Unit": {
+            type: Excel.CellValueType.string,
+            basicValue: product.quantityPerUnit || ""
+        },
+        "Unit Price": {
+            type: Excel.CellValueType.formattedNumber,
+            basicValue: product.unitPrice,
+            numberFormat: "$* #,##0.00"
+        },
+        Discontinued: {
+            type: Excel.CellValueType.boolean,
+            basicValue: product.discontinued || false
+        }
+    },
+    layouts: {
+        // Enter layout settings here.
+    }
+};
+```
```

Comment:
This code is red in GitHub because it's not technically JSON. It's TypeScript that's defining a JSON struct. I'd recommend labelling it TypeScript (or removing the type declaration at the top and labelling it JavaScript).